GW2EIEvtcParser/EncounterLogic/Raids/W8/GreerTheBlightbringer.cs

```diff
@@ -738,16 +756,61 @@ private static void AddRipplesOfRot(NPC target, ParsedEvtcLog log, CombatReplay
             }
         }
+
+        // Blob of Blight - Stationary Orb
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.GreerBlobofBlightStationary, out var blobOfBlight))
+        {
+            foreach (EffectEvent effect in blobOfBlight)
+            {
+                lifespan = effect.ComputeLifespan(log, 7000);
+                replay.Decorations.Add(new CircleDecoration(60, lifespan, Colors.GreenishYellow, 0.3, new PositionConnector(effect.Position)));
+                replay.Decorations.Add(new DoughnutDecoration(60, 80, lifespan, Colors.LightPurple, 0.3, new PositionConnector(effect.Position)));
+            }
+        }
+
+        // Blob of Blight - Moving Orbs
+        // Using the projectile velocity to distinguish between the main orbs and the mini orbs
+        // Main orbs have 0.3 and mini orbs have 0.5, once the main orb hits a player, the mini orbs have 0.4
+        var blobOfBlightMainOrbs = log.CombatData.GetMissileEventsBySkillIDs([BlobOfBlight, BlobOfBlight2, BlobOfBlight3]).Where(x => x.Src == target.AgentItem && x.LaunchEvents.Any(x => x.Speed == 0.3f));
+        var blobOfBlightMiniOrbs = log.CombatData.GetMissileEventsBySkillIDs([BlobOfBlight, BlobOfBlight2, BlobOfBlight3]).Where(x => x.Src == target.AgentItem && x.LaunchEvents.Any(x => x.Speed == 0.4f || x.Speed == 0.5f));
+
+        // Main Orbs
+        foreach (MissileEvent missileEvent in blobOfBlightMainOrbs)
+        {
+            lifespan = (missileEvent.Time, missileEvent.RemoveEvent?.Time ?? log.FightData.FightEnd);
+            for (int i = 0; i < missileEvent.LaunchEvents.Count; i++)
+            {
+                MissileLaunchEvent? launch = missileEvent.LaunchEvents[i];
+                lifespan = (launch.Time, i != missileEvent.LaunchEvents.Count - 1 ? missileEvent.LaunchEvents[i + 1].Time : lifespan.end);
+                if (!launch.TargetedAgent.IsNonIdentifiedSpecies())
+                {
+                    replay.Decorations.Add(new CircleDecoration(60, lifespan, Colors.GreenishYellow, 0.3, new PositionToAgentConnector(launch.TargetedAgent, launch.LaunchPosition, launch.Time, launch.Speed)));
+                    replay.Decorations.Add(new DoughnutDecoration(60, 80, lifespan, Colors.LightPurple, 0.3, new PositionToAgentConnector(launch.TargetedAgent, launch.LaunchPosition, launch.Time, launch.Speed)));
+                }
+            }
+        }
+
+        // Mini Orbs
+        replay.Decorations.AddNonHomingMissiles(log, blobOfBlightMiniOrbs, Colors.GreenishYellow, 0.3, 25);
+    }
+
+    private static void AddEmpoweringBlast(NPC target, ParsedEvtcLog log, CombatReplay replay)
+    {
+        // Empowering Blast - Orbs
+        var orbs = log.CombatData.GetMissileEventsBySrcBySkillIDs(target.AgentItem, [EmpoweringBlast1, EmpoweringBlast2]);
+        replay.Decorations.AddHomingMissiles(log, orbs, Colors.Purple, 0.5, 25);
```

Comment:
Interesting, those are homing?

Reply:
Yeah they change direction, can be noticed when reaching 66% with orbs that are aimed from ereg to gree/reeg as they teleport and the orbs change directions